sql_generators/feature_usage/__init__.py

```diff
@@ -1,15 +1,15 @@
 """Feature usage table generation."""
-import click
 import os
 import shutil
-import yaml
+from pathlib import Path
 
+import click
+import yaml
 from jinja2 import Environment, FileSystemLoader
-from pathlib import Path
 
 from bigquery_etl.cli.query import update
-from bigquery_etl.util.common import write_sql
 from bigquery_etl.format_sql.formatter import reformat
+from bigquery_etl.util.common import write_sql
```

Comment:
I ran `black` on `sql_generators/`, so there are a couple of formatting changes in those files